### PR TITLE
Add get ready post

### PR DESCRIPTION
**Pull request by mxsasha**
To be merged on September 16. Needs to be aligned with me sending the first batch of venueless links.

----
📚 Documentation preview 📚: https://writethedocs-www--2205.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2024/news/get-ready.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Head over to the website and check out the `Welcome Wagon guide <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/welcome-wagon/>`_ to help you get the most out of our event this year.
+
+
+All our events and activities
+-----------------------------
+
+There are many events at the conference beyond the main track of speakers.
+We hope that you'll get involved with some of our other events,
+and get to know more folks that are hanging out at the conference!
+
```
Comment (rosewms) [inline docs/conf/atlantic/2024/news/get-ready.rst:40]: @mxsasha Does it make sense to have Writing Day here with a link to the event and a note that they are welcome to bring a project to announce day of?

Comment (mxsasha) [reply docs/conf/atlantic/2024/news/get-ready.rst:40]: Yes, it belongs in there I think. Will add.

File: docs/conf/atlantic/2024/news/get-ready.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+.. post:: Sept 16, 2024
+   :tags: {{year}}, {{shortcode}}-{{year}}, tickets
+
+Get Ready for the conference!
+=============================
+
+Hey everyone,
+
+We're just a few days from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 3 days of content, collaboration, and conversation.
```
Comment (plaindocs) [inline docs/conf/atlantic/2024/news/get-ready.rst]:
```suggestion
We're just a few days from the start of the conference, and we're getting excited to see you soon! We're looking forward to gathering for three days of content, collaboration, and conversation.
```

File: docs/conf/atlantic/2024/news/get-ready.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+We have a few reminders before we welcome you all on Sunday:
+
+* Ticket sales close this week, so grab yours now!
+* There are lots of ways to get involved at the conference on the day-of, which we list out below so you can prep a bit for them.
```
Comment (plaindocs) [inline docs/conf/atlantic/2024/news/get-ready.rst]:
```suggestion
* There are lots of ways to get involved at the conference on the day-of, which we list out below so you can prepare for them if you'd like to.
```

**Review by plaindocs — COMMENTED** [review]
LGTM

**Pull request merged**